commands/config.go

```diff
@@ -78,6 +78,31 @@ func getValue(path []string, obj interface{}) (interface{}, bool) {
 		return val.Interface(), true
 	} else if val.Kind() == reflect.Struct {
 		return getValue(path[1:], val.Interface())
+	} else if val.Kind() == reflect.Map {
+
+		for i := len(path); i >= 1; i-- {
+			remainingPath := strings.Join(path[1:i], ".")
+
+			mapIter := val.MapRange()
+
+			for mapIter.Next() {
+				if mapIter.Key().String() == remainingPath {
```

Comment:
Why don't use [HasPrefix](https://golang.org/pkg/strings/#HasPrefix) to simplify?